Extensions/SendTo/InstructionsViewController.swift

```diff
@@ -47,6 +47,7 @@ func setupHelpView(view: UIView, introText: String, showMeText: String) {
     label1.lineBreakMode = NSLineBreakMode.ByWordWrapping
     label1.font = InstructionsViewControllerUX.TextFont
     label1.textColor = InstructionsViewControllerUX.TextColor
+    label1.textAlignment = NSTextAlignment.Center
     label1.snp_makeConstraints { (make) -> Void in
         make.width.equalTo(250)
```

Comment:
Not sure what the UI specification is but I believe we had it always fixed to 250 width. The center alignment will center align the text while the 250 width will force the text to be a maximum of 250 px wide. Here's what it looks like on landscape/portrait iPhone 6

![simulator screen shot oct 19 2015 11 08 42 am](https://cloud.githubusercontent.com/assets/434322/10581684/01910d7e-7652-11e5-8cdd-dd524a563dd8.png)
![simulator screen shot oct 19 2015 11 08 44 am](https://cloud.githubusercontent.com/assets/434322/10581687/0458ffee-7652-11e5-8899-f71257cd836e.png)